pynitrokey/cli/nk3/__init__.py

```diff
@@ -712,3 +717,72 @@ def verify(ctx: Context, name: str, code: int, experimental: bool) -> None:
             local_print(
                 f"Device returns error: {e}. This credential id might not be registered, or the provided HOTP code has not passed verification."
             )
+
+
+def ask_for_passphrase_if_needed(app: OTPApp) -> Optional[str]:
+    passphrase = None
+    is_passphrase_set = app.select().challenge is not None
+    if is_passphrase_set:
+        passphrase = prompt("Current Password", hide_input=True)
```

Comment:
Could you add support for setting the password with a environment variable?

Reply:
Good point. I've used the AskUser helper, and noticed that it does not suggest using the env variable, or show it's name to user anywhere. Probably worth updating in the following PRs. Will add it to the help screen for now.